### ISSUE TITLE
Dictionary - Suffixes improvement

### ISSUE THREAD

**Issue by bormachine** [blchinezu/pocketbook-coolreader#207]
When using text selection with dictionary, if word is not matched in dictionary- auto-remove suffixes until match is found. Mostly encountered issues with: -s, -ly, -ing, -ed, but also  -ment, -ness and other.

Some dictionaries will refer to the original word without the suffix. Clicking on the word within the dictionary window/area will also be good. This is already in #76

**Comment by blchinezu** [comment]
This could be done but it should be implemented in the dictionary. It wouldn't be the best thing to hard code this stuff since it's valid only for english.

**Comment by spcmd** [comment]
Yes, I really hope this can be made, it would be a great help when using the dictionary.

**Comment by baltov** [comment]
At InkPad 3, FW5, if i change row 3815, in cr3pocketbook.cpp, 
`_translateResult = LookupWordExact((char *)what.c_str(), &word, &translation);`
with
`_translateResult = LookupWord((char *)what.c_str(), &word, &translation);`

dictionary work as expected. Maybe LookupWordExact return something not 0?
How can I  enable trace log?
